src/EditorFeatures/VisualBasicTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.vb

```diff
@@ -220,5 +220,44 @@ Partial Class Outer
 End Class"
             Await TestMoveTypeToNewFileAsync(code, codeAfterMove, expectedDocumentName, destinationDocumentText)
         End Function
+
+        <WorkItem(17879, "https://github.com/dotnet/roslyn/issues/17879")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveType)>
+        Public Async Function DoNotMoveLeadingFileTrivia() As Task
+            Dim code =
+"
+''' <summary>
```

Comment:
Having a banner and a class with a comment on it without namespaces/usings works as expected:

```
// Copyright banner for something

/// <summary>A doc comment</summary>
class C { }
```